trimChamber.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python
+#!/bin/env python2.7
```

Comment:
Again, this file is obsoleted by confChamber.py. Do we really care? I do think we should try to write code which is compatible with python and does not require python2.7 in general though.